test/Case Insensitive Text.coffee

```diff
@@ -0,0 +1,6 @@
+helpers = require './helpers'
+
+helpers.describe 'Case Insensitive Text', (test) ->
+	string = 'hello world'
+	describe 'validate', ->
+		test.validate(string, true, string)
```

Comment:
No, because the string should be unchanged. `validate` just makes sure it's a valid input and does any manipulation that is needed, which in this case is just checking it's a valid string - it's any comparisons against the string that should be case insensitive, but the string itself should return its casing when it's returned, eg if I name my app `SenseSnake` then searching for `sensesnake`, `SeNsEsNaKe`, etc should find it, but I should still get the result as `SenseSnake`